.circleci/config.yml

```diff
@@ -68,7 +71,11 @@ workflows:
             parameters:
               ruby_version:
                 - 2.6.6
+                - 2.7.1
+                - 2.7.2
                 - 2.7.3
+                - 2.7.4
+                - 2.7.5
               elasticsearch_version:
                 - 6.8.15
                 - 7.10.1
```

Comment:
Should this not match what's in the others?  I think the bot_detection repo lists a higher version for ES here